docs/compatible_configs.md

```diff
@@ -19,7 +19,18 @@ Compatible configuration files can be
 _Black_ also formats imports, but in a different way from isort's defaults which leads
 to conflicting changes.
 
-### Configuration
+### Profile
+
+Since version 5.0.0, isort supports [profiles](https://pycqa.github.io/isort/docs/configuration/profiles/) to allow easy interoperability with common code styles. You can set the black profile in `pyproject.toml`:
```

Comment:
It _may_ be worth clarifying that the isort's profile can be configured via any of its supported configuration files, not just `pyproject.toml`. But maybe it's just me who sees how this could go wrong.

Reply:
Good point :+1: I've re-phrased it and added a link to the [Supported Config Files](https://pycqa.github.io/isort/docs/configuration/config_files/) in the isort docs.